charts/wharf-cmd/values.yaml

```
@@ -126,21 +127,22 @@ provisioner:
   # [Read more (kubernetes.io)](https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/)
   extraEnvs: []
 
-  # -- (string[]) Arguments to add to the container. E.g `[ "--another-arg", "value" ]`
+  # -- Arguments to add to the container. E.g `[ "--another-arg", "value" ]`
   extraArgs: []
 
   # -- Configuration for wharf-cmd added only to the aggregator component.
```

missed post-copypaste edit
```suggestion
  # -- Configuration for wharf-cmd added only to the provisioner component.
```